### ISSUE TITLE
`android_semantics_integration_test` is failing `AccessibilityBridge Popup Controls Popup Menu has correct Android semantics`: Unexpected: {AndroidSemanticsAction.accessibilityFocus, AndroidSemanticsAction.longClick}

### ISSUE THREAD

**Issue by jmagman** [flutter/flutter#117479]
Started failing at https://github.com/flutter/flutter/pull/117338.
```
[android_semantics_integration_test] [STDOUT] stdout: [ +194 ms] 00:15 [32m+6[0m[31m -1[0m: AccessibilityBridge Popup Controls Popup Menu has correct Android semantics [1m[31m[E][0m[0m
[android_semantics_integration_test] [STDOUT] stdout: [        ]   Expected: AndroidSemanticsNode with className: android.widget.Button with actions: [AndroidSemanticsAction.click] with flag isChecked: false with flag isEnabled: true with flag isFocusable: true
[android_semantics_integration_test] [STDOUT] stdout: [        ]     Actual: AndroidSemanticsNode:<{flags: {isEditable: false, isEnabled: true, isDismissible: false, isCheckable: false, isChecked: false, isFocused: false, isLongClickable: true, isFocusable: true, isPassword: false}, id: 33, text: null, rect: {height: 144, width: 144, bottom: 360.0, top: 312.0, left: 77.0, right: 125.0}, actions: [16, 32, 64], className: android.widget.Button, liveRegion: 0, contentDescription: 
[android_semantics_integration_test] [STDOUT] stdout: [        ]             Show menu}>
[android_semantics_integration_test] [STDOUT] stdout: [        ]      Which: Expected actions: [AndroidSemanticsAction.click]
[android_semantics_integration_test] [STDOUT] stdout: [        ]             Actual actions: [AndroidSemanticsAction.accessibilityFocus, AndroidSemanticsAction.click, AndroidSemanticsAction.longClick]
[android_semantics_integration_test] [STDOUT] stdout: [        ]             Unexpected: {AndroidSemanticsAction.accessibilityFocus, AndroidSemanticsAction.longClick}
[android_semantics_integration_test] [STDOUT] stdout: [        ]             Missing: {}
```

This shard was already marked flaky due to https://github.com/flutter/flutter/issues/114360 but now it is always failing.

@harperl-lgtm and @chunhtai
